app/src/main/java/org/mozilla/focus/ScreenController.kt

```diff
@@ -23,10 +23,9 @@ object ScreenController {
      * Loads the given url. If isTextInput is true, there should be no null parameters.
      */
     fun onUrlEnteredInner(context: Context, fragmentManager: FragmentManager,
-                          sessionManager: SessionManager,
                           urlStr: String,
-                          isTextInput: Boolean, autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?,
-                          inputLocation: UrlTextInputLocation?) {
+                          isTextInput: Boolean,
+                          autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?, inputLocation: UrlTextInputLocation?) {
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.

Reply:
Sputnik isn't reading our detektConfig, unfortunately, but this doesn't fail locally.